### ISSUE TITLE
codegen bugs

### ISSUE THREAD

**Issue by snikeguo** [mamift/LinqToXsdCore#71]
[din_spec.zip](https://github.com/user-attachments/files/18063775/din_spec.zip)
run:
```
inqToXsd.exe gen  .  -o test
```

The generated C # code only contains a few types, and a large number of types are lost.such as:SessionSetupReqType...

**Comment by mamift** [comment]
Yes this is a bug, but it is possible to work around by specifying each XSD manually:

```
linqtoxsd config -e .\V2G_CI_AppProtocol.xsd; 
linqtoxsd gen -a .\V2G_CI_AppProtocol.xsd
```

and so on.
